### DISCUSSION TITLE
Lampadina HUE senz HUB Philips

### DISCUSSION THREAD

**Discussion by riccionese** [indomus/forum#1309] [ZigBee, Z-Wave, Thread, Matter]
Cia a tutti, 
nel mio HA da ormai un anno ho integrato, con l'HAT di Giovanni su un Pi0 dedicato, diversi dispositivi Zigbee Xiaomi, Tuya, Sonoff ed **_una lampadina HUE_**  e con Zigbee2mqtt tutto funziona alla meraviglia.
In realtà fino a circa Natale ha funzionato tutto ma poi (non so di preciso quando e non so da cosa può essere dipeso) la lampadina HUE_Bulb di Philips ha iniziato a fare le bizze.
Non accetta i comandi oppure li accetta ... minuti dopo aver impartito il comando (quando l'accetta)
Ho controllato tutto quello che mi è venuto in mente di controllare, compresi il log di Z2M ma non trovo molte info che mi facciano capire cosa non va.
Questo la traccia nel log di un errore a seguito di un comando On/Off:
```
error 2022-01-12 16:44:06: Publish 'set' 'state' to 'Bulb_HUE' failed: 'Error: Command 0x0017880106e11e60/11 genLevelCtrl.moveToLevelWithOnOff({"level":188,"transtime":0}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Data request failed with error: 'MAC no ack' (233))' info  2022-01-12 16:44:06: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to 'Bulb_HUE' failed: 'Error: Command 0x0017880106e11e60/11 genLevelCtrl.moveToLevelWithOnOff({\"level\":188,\"transtime\":0}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null,\"writeUndiv\":false}) failed (Data request failed with error: 'MAC no ack' (233))'","meta":{"friendly_name":"Bulb_HUE"},"type":"zigbee_publish_error"}'
```

Dal sensore "update available" vedevo che ce n'era uno disponibile ma con l'interfaccia WEB di Z2M non accettava l'aggiornamento OTA

allora ho eliminato la lampadina da Z2M, l'ho agganciata con l'app philips, ho fatto l'aggiornamento.

Poi, come avevo fatto la prima volta (sono abbastanza convinto di ricordarmi bene) ho attivato il pairing su ZB ed ho "rimosso" la lampadina dall'app Philips.
Solo che non c'è verso che la HUE_Bulb riappaia in Z2M...

Ho cercato "**_MAC no ack' (233)_**", unica informazione sull'errore ed ho trovato diversi thread ma non sono riuscito a ricondurre alla mia casistica

qualcuno ha delle dritte ?

p.s.: sia HA che Z2M sono all'ultima rel disponibile

**Comment by maxcanna** [comment]
Hai altri dispositivi nella rete? Prova a spegnerli mentre tenti il join

**Reply by riccionese** [reply]
si. come detto ho altri aggeggi ZB.
e la lampadina HUE è l'unica sotto 220. Il resto è tutto wifi. 
dici che spegnerli conta ?

**Reply by snakuzzo** [reply]
@maxcanna parlava di altri dispositivi "nella rete zigbee"

**Reply by riccionese** [reply]
Probabilmente non ho capito cosa si intenda (e può certamente essere :-) )  per "rete zigbee" oppure è quello che ho risposto: ho altri dispositivi ZB collegati allo stesso coordinator (l'HAT di Giovanni) ma tutti "a batteria"

**Reply by maxcanna** [reply]
Fai questa prova. Crea un'altra rete temporanea con differente canale, pan id e key. Vedi se la lampadina entra nella rete. Così escludi malfunzionamenti. Espelli la lampadina dalla rete e riattiva la vecchia rete. La lampadina dovrebbe entrare nella rete originale

**Reply by riccionese** [reply]
non so se ho capito ma se riesco, stasera quando torno a casa, ci provo .... 
intanto grazie
